google/cloud/bigquery/v2/minimal/internal/job_request_test.cc

```diff
@@ -161,10 +161,16 @@ TEST(ListJobsRequestTest, Success) {
   expected.AddQueryParameter("maxResults", "10");
   expected.AddQueryParameter(
       "minCreationTime",
-      internal::FormatRfc3339(request.min_creation_time().value()));
+      std::to_string(std::chrono::time_point_cast<std::chrono::milliseconds>(
```

Comment:
Here and below, can you add static values instead of calling the same function? This makes it easy to review the value that is expected. 

This should have been done for the RFC change as well.

Reply:
I added static variables.

I also encapsulated the logic of casting from time_point to int. I left it in this file as I didn't find other places of using such functionality and already created 'job_util' file. But I can move it to another file, if you point me the best option.